jsx/main.jsx

```diff
@@ -1,7 +1,7 @@
 
 import React from 'react'
 import ReactDOM from 'react-dom'
-import Hello from 'hello'
+import Hello from './hello'
```

Comment:
Not sure if this was technically 'required', but it's good practice to specify the relative path otherwise it looks like a package import